cardano-api/src/Cardano/Api/Fees.hs

```diff
@@ -352,7 +352,7 @@ data ScriptExecutionError =
        -- the whole point of it is to discover how many execution units are
        -- needed).
        --
-     | ScriptErrorEvaluationFailed Plutus.EvaluationError
+     | ScriptErrorEvaluationFailed Plutus.EvaluationError [Text.Text]
```

Comment:
yes indeed

Reply:
though I didn't attempt to display them yet, this started out as a WIP to get a version of node out sooner rather than later. https://github.com/input-output-hk/cardano-node/pull/3302/commits/e7bea2d714dc9f2f280f673caddffb2138aeadaf#diff-12e0bb9f3db22ac71c93ed3932f54c91a6f6bf7243afaa29ccd7ecb3657e6069R385